Cargo.toml

```diff
@@ -56,8 +56,8 @@ quickcheck = { version = "1" }
 quickcheck_macros = { version = "1" }
 quote = { version = "1" }
 scale = { package = "parity-scale-codec", version = "3.7.4", default-features = false, features = ["derive"] }
-scale-decode = { version = "0.14.0", default-features = false }
-scale-encode = { version = "0.8.0", default-features = false }
+scale-decode = { version = "0.16.0", default-features = false }
```

Comment:
Required due to dependency clashes if not otherwise bumped to the same versions used by subxt.